deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -121,26 +121,33 @@ instancetype=c5.4xlarge
 deploytriplet=None
 
 # Golden Gate Multi-Cycle RAM Optimization Demo Recipes
-
+# RAM Optimizations enabled by adding _MCRams PLATFORM_CONFIG string
 [firesim-quadcore-no-nic-ddr3-llc4mb-gg]
 DESIGN=FireSimNoNIC
-TARGET_CONFIG=FireSimRocketChipQuadCoreConfig
-PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig_MCRams_F90MHz
+TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimRocketChipQuadCoreConfig
+PLATFORM_CONFIG=BaseF1Config_MCRams_F90MHz
 instancetype=c5.4xlarge
 deploytriplet=None
 
 [fireboom-singlecore-no-nic-ddr3-llc4mb-gg]
 DESIGN=FireBoomNoNIC
-TARGET_CONFIG=FireSimBoomConfig
-PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig_MCRams_F90MHz
+TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimBoomConfig
+PLATFORM_CONFIG=BaseF1Config_MCRams_F90MHz
 instancetype=c5.4xlarge
 deploytriplet=None
 
 # SHA-3 Demo Recipes
 
 [firesim-singlecore-sha3-l2-nic-ddr3-llc4mb]
 DESIGN=FireSim
-TARGET_CONFIG=FireSimRocketChipSha3L2Config
-PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig_F90MHz
+TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimRocketChipSha3L2Config
+PLATFORM_CONFIG=BaseF1Config_F90MHz
+instancetype=c5.4xlarge
+deploytriplet=None
+
+[firesim-singlecore-sha3-l2-nic-ddr3-llc4mb-120mhz]
+DESIGN=FireSim
+TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimRocketChipSha3L2Config
+PLATFORM_CONFIG=BaseF1Config_F120MHz
```

Comment:
Should we simply target 120 MHz by default?  The AGFI build took slightly less than 4 hours for me on a z1d.xlarge instance.